docs/src/testing/basics.md

```diff
@@ -0,0 +1,80 @@
+# Testing Basics
+
+If you're new to Rust, you'll want to review these important tools to help you build tests.
+
+## The `assert!` macro
+
+You can use the `assert!` macro to enforce certain outcomes in your test. This macro throws an error and fails the test if the expression inside evaluates to `false`.
+
+```rust, ignore
+assert!(value == 5);
+```
+
+## The `assert_eq!` macro
+
+The `assert_eq!` macro works a lot like the `assert` macro, however instead it accepts two values, and throws and error if those values are not equal.
+
+```rust, ignore
+assert_eq!(balance, 100);
+```
+
+## The `assert_ne!` macro
+
+The `assert_ne!` macro works just like the `assert_eq!` macro, but it will throw an error if the two values are equal.
+
+```rust, ignore
+assert_ne!(address, 0);
+```
+
+## The `println!` macro
+
+You can use the `println!` macro to print values to the console.
```

Comment:
Technically the output goes to `stdout` which may or may not be a tty. 

But I'm not sure if being technically accurate serves any purpose here besides guarding against bureaucrats :D.